src/Conference/ConferenceItem.test.jsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import { MemoryRouter } from "react-router-dom";
+import ConferenceItem from "./ConferenceItem";
+
+test("displays conference item", async () => {
+  const mockConference = {
+    id: 1,
+    name: "Mock Conference",
+    topic: "Mock Topic",
+    dateTime: "2020-11-12T12:34:11Z",
+    city: "Mock City"
+  };
+
+  render(
```

Comment:
I'm not sure what screen is as I haven't used this library that much, but When I have screen is not needed. You can access `getByRole` etc from the render method. i.e. `const { getByText, getByRole } = render( <SomeThingToRender />)`

Reply:
Seems to be just another way of doing the same thing you just described as far as I can tell, I prefer your way so I've gone with that.